### ISSUE TITLE
definition of energy

### ISSUE THREAD

**Issue by JohnDenker** [ericmazur/PnPbook#2]
On page 120, in the glossary for chapter 5, there appears a definition of "Energy".

1) As a point of style, it is a dubious practice to introduce new information in a section that is supposed to be just a summary.  It would be better to discuss the definition (or non-definition) in the body of the chapter.  The glossary can give a summary, and point to the fuller discussion.

2) The existing "definition" is not very informative.  Somebody who already knows what energy is will agree that the words are "consistent" with reality ... whereas somebody who doesn't already understand won't be able to figure it out.

3) The "definition" is only _weakly_ consistent with reality, insofar as it is equally consistent with salinity and energy density and all sorts of other things that aren't really energy.

4) It is far more important to understand what energy _does_ than to rote-memorize a pithy dictionary-style definition of what energy "is".  

 This is a problem because all-too-often, incoming students have been trained to place far too much emphasis on pithy definitions and the rote regurgitation thereof.  The point needs to be made that in the real world, words derive their meaning _from how they are used_ -- not from some dictionary-style definition.  This will come as a shock to some students, but it's true and important.

In other words:  Energy is as energy does.  

Also: The conservation law is a central part of any understanding of what we mean by energy.

5) As for energy in particular, I would emphasize that energy is so fundamental that it cannot be defined in terms of anything more fundamental.  It cannot be defined _a priori_.   Instead, it is defined implicitly, retroactively, and recursively.   The definition starts with some well-known forms of energy, and then is extended as we encounter new forms.  The new forms of energy are recognized as such by the role they play in the conservation law.
